Jenkinsfile

```diff
@@ -964,18 +964,6 @@ try {
         }
       }
     },
-    'Caffe GPU': {
```

Comment:
Please comment this out and add a comment above with a link to the Github issue that tracks this effort. This allows us to grep for GitHub links to find open issues in our code.